marimo/_messaging/streams.py

```diff
@@ -45,7 +51,7 @@
 STD_STREAM_MAX_BYTES = int(os.getenv("MARIMO_STD_STREAM_MAX_BYTES", 1_000_000))
 
 
-class Stream:
+class ConcurrentStream(Stream):
```

Comment:
Instead of `ConcurrentStream`, how about something more descriptive like `PipeStream` or `ThreadSafeStream`? Same for Stdout/Stderr

Reply:
fixed